examples/src/main/scala/Protocol.scala

```diff
@@ -30,7 +30,8 @@ import codecs._
 @GenClient(remotely.Protocol.empty.codec[Int].specify1[Int,Int]("fac").signatures)
   object FacClient
 
-@GenServer(remotely.Protocol.empty.codec[List[Int]]) abstract class Foo
+// TODO(ahjohannessen): @stew <- Compiling Foo fails somehow?
+// @GenServer(remotely.Protocol.empty.codec[List[Int]]) abstract class Foo
```

Comment:
@timperrett It might be that something breaks wrt `GenServer`. I can see that it imports everything that is inside `remotely.codecs` when it generates actual abstract class.